ui/app/components/ui/token-input/token-input.component.js

```diff
@@ -80,6 +80,8 @@ export default class TokenInput extends PureComponent {
 
     const multiplier = Math.pow(10, Number(decimals || 0))
     const hexValue = multiplyCurrencies(decimalValue || 0, multiplier, {
+      multiplicandBase: 16,
```

Comment:
The multiplicand appears to be a decimal value here, not a hex value (the variable `decimalValue`).